src/App.tsx

```diff
@@ -1,46 +1,100 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import "./App.scss";
 import CreateToDo from "./components/CreateToDo";
 import Header from "./components/Header";
 import Todo from "./components/Todo";
 
+interface itemsTypes {
+  todoId: number;
+  content: string;
+  isChecked: boolean; //not working yet
+  isEmpty: boolean;
+}
+
 const App = () => {
-  const [items, setItems] = useState<(string | null)[]>([
-    "Item 1",
-    "Item 2",
-    "Item 3",
+  const [items, setItems] = useState<itemsTypes[]>([
+    {
+      todoId: 1,
+      content: "Item 1",
+      isChecked: false,
+      isEmpty: false,
+    },
+    {
+      todoId: 2,
+      content: "Item 2",
+      isChecked: false,
+      isEmpty: false,
+    },
+    {
+      todoId: 3,
+      content: "Item 3",
+      isChecked: false,
+      isEmpty: false,
+    },
   ]);
 
-  const [hasNull, setHasNull] = useState(false);
+  console.log(items);
+
+  const getHighestId = () => {
+    const itemsId = items.map((item) => item.todoId);
+    return Math.max(...itemsId) + 1;
```

Comment:
When we delete all todo's and try to create a new one it will be spreading an empty array. 
Something like this:
`Math.max(...[])`
The result for this will be `-Infinity` which will cause all todo items to have the same ID (-Infinity).
To fix this, you should ensure that itemsId has at least one value to perform `Math.max(...itemsId)` function. If there is no value it means it is the first item and then you should return the value `1`
Would be something like this:
```suggestion
    return itemsId.length ? Math.max(...itemsId) + 1 : 1;
```

Reply:
Understood! I already made the changes.